ui/lib/nativeUtils.js

```diff
@@ -0,0 +1,14 @@
+import { IPC_CLIPBOARD_WRITETEXT, IPC_DIALOG_SHOWOPENDIALOG } from "./types.js";
```

Comment:
The `Utils` suffix seems unnecessary, this can be all `native` library functionality, which inherently should provide utility. :>